README.md

```diff
@@ -60,6 +60,10 @@ Current supported and tested models are `Llama2` (7 and 13B configurations have
 # if you want to use one GPU on multi-gpu machine
 export CUDA_VISIBLE_DEVICES=0
 
+MODEL_PATH=llama-7b-hf # Huggingface model id or path to a checkpoint
+DATA_PATH=twitter_complaints.json # Path to the dataset
+OUTPUT_PATH=out # Path to the output folder where the checkpoints are saved
+
```

Comment:
```suggestion
```
suggest to now have these extra arguments here, since this README is for generic use case

also `llama-7b-hf` is not the official huggingface name actually,

Reply:
If your intent is to avoid users from directly executing as it is, and force them to read and set variables themselves, we can comment out the values like below, instead of deleting:

```
# Please set the environment variables:
# MODEL_PATH = llama-7b-hf # Huggingface model id or path to a checkpoint
# TRAIN_DATA_PATH=twitter_complaints.json # Path to the train dataset
# OUTPUT_PATH=out # Path to the output folder where the checkpoints are saved
```
I think its good to have it in README though, as its just an example demonstrating that we need to set these environment variables and roughly what their values look like. Users trying out the repository may not understand `data_path ` accepts a single json for example - we have explained how data should be formatted directly above in README, but good to show example that it accepts a single JSON

Reply:
I agree its in the prompt tuning one also. If this is fixed, it needs to be fixed in both places. I leave it to @Ssukriti to decide which version she wants.

Reply:
You can change the name of the model as you want , I just meant to include it as example . You can use the meta one

Reply:
Updated @Ssukriti !